SourceEditorCompose/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.microsoft.device.display.samples.sourceeditorcompose">
+
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Do you need this permission? afaik you are just rendering a local `.html` file, right?

Reply:
Yep, removed it!